traffic_ops/app/db/migrations/2022041810412200_cdni_capacity_limit_with_scopes.down.sql

```diff
@@ -0,0 +1,51 @@
+/*
```

Comment:
The timestamps of these migrations need to be changed to be after what we have in master currently.